tss-esapi-sys/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tss-esapi-sys"
+version = "0.1.0"
+authors = ["Ionut Mihalcea <ionut.mihalcea@arm.com>",
+           "Hugues de Valon <hugues.devalon@arm.com>"]
```

Comment:
Yes agree that it would be nice to put the same mention that in our copyright files, "Contributors to the Parsec project" or something similar. We should, by the way, do that everywhere!